tests/scripts/all.sh

```diff
@@ -6050,6 +6050,17 @@ component_test_tls13_no_compatibility_mode () {
     tests/ssl-opt.sh
 }
 
+component_test_default_minus_session_tickets() {
```

Comment:
My instinct would be to go for full minus session tickets, rather than default. Though I'm not sure that makes much of a difference now, so I won't insist on it if you disagree.

Reply:
No I do not have any objections to that, changed. The main difference is probably USE_PSA_CRYPTO being enabled or not.